### PR TITLE
Line break

### PR DESCRIPTION
**Pull request by jburel**
In this wrap, we fix the text wrapping
 see https://trac.openmicroscopy.org.uk/ome/ticket/12890

To test:
 * log as a group owner e.g. user-1
 * Select a project
 * Click delete
 * Check that the text in red is wrapped

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/util/ui/UIUtilities.java [hunk 1]
```diff
@@ -958,6 +974,44 @@ public static JLabel setTextFont(String s, int fontStyle)
         label.setFont(newFont);
         return label;
     }
+
+    /**
+     * Displays the specified string into a {@link JLabel} and sets 
+     * the font to <code>bold</code>.
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/UIUtilities.java]: Doesn't it give a choice of style?

**Comment by jburel** [comment]
@mtbc: doc improved

**Comment by gusferguson** [comment]
@jburel 

Tested using OMERO.insight-5.1.2-194-f639901-ice34-b544-mac_Java6 user-6 private-1

Text is now wrapping, but the label/text field is remaining as a single line, so text is illegible.

confirm delete

**Comment by jburel** [comment]
humm, I did not have that issue.

**Comment by gusferguson** [comment]
@jburel - it looks OK in Windows.

parallels desktop

**Comment by gusferguson** [comment]
@jburel - and OK with Java 7 version - OMERO.insight-5.1.2-194-f639901-ice35-b544-mac_Java7+

confirm delete

**Comment by mtbc** [comment]
On Linux / OpenJDK I get,
dialog

**Comment by gusferguson** [comment]
Grief!!

**Comment by jburel** [comment]
@gusferguson: last commit should work for 1.6+

**Comment by gusferguson** [comment]
@jburel 

OMERO.insight-5.1.2-233-27f65e6-ice34-b545-mac_Java6
- layout fine for both own and other’s data and when there are annotations - see screenshots

OMERO.insight-5.1.2-233-27f65e6-ice34-b545-mac_Java7+
OMERO.insight-5.1.2-233-27f65e6-ice34-b545-win
- both behave as expected for all types

Good to merge.


confirm delete

confirm delete

confirm delete

**Pull request merged**